README.md

```diff
@@ -275,10 +294,11 @@ The **detailed_summary.tsv** output file generated by `staramr` contains the fol
 * __Data Type__: The type of gene it is either a **Resistance** gene or a **Plasmid** gene
```

Comment:
Could you move **Data Type** next to **Data** (to match order of columns).

Also, could you change the description to be something like:

>The type of gene (**Resistance** or **Plasmid**), or **MLST**.

Reply:
Fixed in my next commit